src/Language/PureScript/Docs/Types.hs

```diff
@@ -184,6 +185,17 @@ data DeclarationInfo
 
 instance NFData DeclarationInfo
 
+-- |
+-- Wraps enough information to properly render the kind signature
+-- of a data/newtype/type/class declaration.
+data KindInfo = KindInfo
+  { kiKindSigFor :: P.KindSignatureFor
+  , kiType :: Type'
```

Comment:
This is really a kind rather than a type, right? Maybe this field should be called `kiKind`?

Reply:
Yeah, it is. I'll make the change. I think I'll also update other usages of the underlying value to say kind rather than type.

Reply:
Sounds good, thanks!